alchemy/test/aws/control/proxy.test.ts

```diff
@@ -0,0 +1,166 @@
+import { describe, expect } from "bun:test";
+import { alchemy } from "../../../src/alchemy.js";
+import AWS from "../../../src/aws/control";
+import { destroy } from "../../../src/destroy.js";
+import { BRANCH_PREFIX } from "../../util.js";
+// must import this or else alchemy.test won't exist
+import "../../../src/test/bun.js";
+
+const test = alchemy.test(import.meta);
+
+describe("AWS Cloud Control Proxy", () => {
+  const testId = `${BRANCH_PREFIX}-test-bucket-${Date.now()}`;
```

Comment:
Do not use Date.now(). Each run should be deterministic and idempotent. This will create sporadic buckets. 

Instead, make sure the test's `finally` destroys the test scope. 

We should also add an adopt: boolean property to a CC resource that can be used to adopt the bucket if it already exists on first creation.

Reply:
The creation failed although the last bucket with the same name was deleted

I'll try and see if I can reproduce that tomorrow.

Regarding the adopt boolean, sounds good to me